website/src/pages/docs/integrations/integration-with-graphos.mdx

```diff
@@ -0,0 +1,75 @@
+---
+description:
+  Apollo GraphOS is a cloud platform for managing a supergraph. Most notably, it can be used as a
+  schema registry and usage reporting tool.
+---
```

Comment:
We already have https://the-guild.dev/graphql/yoga-server/docs/features/apollo-federation.

It feels like this page should be integrated within that page?

Now we document how to us GraphOS as a schema registry, but we don't document how to use Hive as a schema registry... 🤔 Hive is basically documented via the Hive Docs, but we heavily prioritize using apollo router there in the meantime. So to me it feels like we should cover both registries here in the meantime.


Also "Integrations" were meant for runtimes/frameworks, I feel like being a federation router is more a Feature?

Reply:
Yes, you are totally right, I've missed this section, I will move the documentation.

And yes, we should provide a good Hive integration in the yoga doc too. It can even be a remote markdown if needed, to ease its update ?

Reply:
By the way, we should probably recommend `@graphql-tools/federation` instead of our old apollo-federation plugin. This would remove the dependency on Apollo's gateway and the Elastic Licence that comes with it.